src/VisualStudio/Xaml/Impl/Implementation/LanguageServer/Extensions/SymbolExtensions.cs

```diff
@@ -53,16 +52,51 @@ public static async Task<IEnumerable<TaggedText>> GetDescriptionAsync(this ISymb
                 return Enumerable.Empty<TaggedText>();
             }
 
-            var textContentBuilder = new List<TaggedText>();
-            textContentBuilder.AddRange(symbol.ToDisplayParts(s_descriptionStyle).ToTaggedText());
+            var description = await symbolDisplayService.ToDescriptionPartsAsync(codeProject.Solution.Workspace, semanticModel, 0, ImmutableArray.Create(symbol), SymbolDescriptionGroups.MainDescription, cancellationToken).ConfigureAwait(false);
+
+            var builder = new List<TaggedText>();
+            builder.AddRange(description.ToTaggedText());
 
-            var documentation = symbol.GetDocumentationParts(semanticModel, offset, formatter, cancellationToken);
+            var documentation = symbol.GetDocumentationParts(semanticModel, 0, formatter, cancellationToken);
             if (documentation.Any())
             {
-                textContentBuilder.AddLineBreak();
-                textContentBuilder.AddRange(documentation);
+                builder.AddLineBreak();
+                builder.AddRange(documentation);
+            }
+
+            var remarksDocumentation = symbol.GetRemarksDocumentationParts(semanticModel, 0, formatter, cancellationToken);
```

Comment:
i'm surprpised you need to write this yourself.  if you use ToDescriptionGroupsAsync, you should have a documentation group already setup for you that you can use.  does that not work?

Reply:
Nope. I would have expected ToDescriptionPartsAsync with SymbolDescriptionGroups.All to give me everything but it does not, so I limited it to MainDescription and called the rest myself.